packages/node-red-s1seven-api/lib/hash/hash.js

```diff
@@ -21,9 +23,7 @@ module.exports = function (RED) {
     node.on('input', async (msg, send, done) => {
       let certificate = msg.payload || globalContext.get('certificate');
       const accessToken =
-        msg.accessToken ||
-        apiConfig?.accessToken ||
-        globalContext.get('accessToken');
+        msg.accessToken || globalContext.get('s1sevenAccessToken');
```

Comment:
this `s1sevenAccessToken` string should be a constant as well.
i'm surprised you didn't ear me whisper this to your brain when you wrote it.

Reply:
Me too! 👍🏼